tests/wicked/startandstop_sut.pm

```diff
@@ -79,9 +79,20 @@ sub run {
     assert_script_run("rm /etc/sysconfig/network/ifcfg-$iface $config $dummy");
     assert_script_run("wicked ifreload --timeout infinite all");
     my $res = script_run("ip link|grep 'dummy0\|br0'");
-    $results{3} = $res ? "PASSED" : "FAILED";
+    $results{3} = $res != 0 ? "PASSED" : "FAILED";
```

Comment:
```suggestion
    $results{3} = $res ? "FAILED" : "PASSED";
```

Reply:
this should be

    $results{3} = $res ? "PASSED" : "FAILED";